Script to fetch all the wifi passwords

rahulkarda opened the pull request:
## Description
This is a simple script that fetches all the saved wifi passwords and prints them into the terminal.

Fixes #345 

## Dependencies

## Future Improvements

Generating a QR code to easily connect mobile devices.

## Mentions

## Screenshots of relevant screens

## Developer's checklist

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-check on my work

**If changes are made in the code:**

- [x] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [x] My changes in code generate no new warnings
- [ ] My changes are breaking another fix/feature of the project
- [ ] I have added test cases to show that my feature works
- [ ] I have added relevant screenshots in my PR

bartick commented on scripts/wifi_password_checker/script.py:
```diff
@@ -0,0 +1,49 @@
+for i in profiles:
+    try:
+
+        results = subprocess.check_output(['netsh', 'wlan', 'show', 'profile', i, 'key=clear']).decode('utf-8',
```
None of the profile exists. Unable to find any profile on my device. 
I am using a m1 macbook air

rahulkarda replied:
@bartick you tried the command "netsh wlan show profile"?

bartick commented on scripts/wifi_password_checker/script.py:
```diff
@@ -0,0 +1,49 @@
+        print("{:<30}|  {:<}".format(i, "ENCODING ERROR"))
+        print("-" * 50)
+
+pswd_txt.close() 
```
Didn't quite get how it is generating the QR

rahulkarda commented:
Yeah I have to add the QR feature. For now it only shows the saved wifi
passwords in the terminal.

On Tue, 11 Oct, 2022, 3:13 pm Bartick Maiti, ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In scripts/wifi_password_checker/script.py
> <https://github.com/metafy-social/daily-python-scripts/pull/349#discussion_r992091451>
> :
>
> > +pswd_txt = open(f"{os.getlogin()}-{platform.node()}", "w")
> +
> +data = subprocess.check_output(['netsh', 'wlan', 'show', 'profiles']).decode('utf-8', errors="backslashreplace").split(
> +    '\n')
> +profiles = [i.split(":")[1][1:-1] for i in data if "All User Profile" in i]
> +
> +pswd_txt.write(f'{"{:<30}|  {:<}".format("WIFI", "Password")} \n')
> +pswd_txt.write(f'{"-" * 50} \n')
> +
> +print("{:<30}|  {:<}".format('WIFI', 'Password'))
> +print("-" * 50)
> +
> +for i in profiles:
> +    try:
> +
> +        results = subprocess.check_output(['netsh', 'wlan', 'show', 'profile', i, 'key=clear']).decode('utf-8',
>
> None of the profile exists. Unable to find any profile on my device.
> I am using a m1 macbook air
> ------------------------------
>
> In scripts/wifi_password_checker/script.py
> <https://github.com/metafy-social/daily-python-scripts/pull/349#discussion_r992092360>
> :
>
> > +        except IndexError:
> +
> +            pswd_txt.write(f'{"{:<30}|  {:<}".format(i, "")} \n')
> +            pswd_txt.write(f'{"-" * 50} \n')
> +            print("{:<30}|  {:<}".format(i, ""))
> +            print("-" * 50)
> +
> +
> +    except subprocess.CalledProcessError:
> +
> +        pswd_txt.write(f'{"{:<30}|  {:<}".format(i, "ENCODING ERROR")} \n')
> +        pswd_txt.write(f'{"-" * 50} \n')
> +        print("{:<30}|  {:<}".format(i, "ENCODING ERROR"))
> +        print("-" * 50)
> +
> +pswd_txt.close()
>
> Didn't quite get how it is generating the QR
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/metafy-social/daily-python-scripts/pull/349#pullrequestreview-1137151812>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASFMWP3WPGRS4H6HQBMUJTLWCUZCPANCNFSM6AAAAAARB6DOTY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
